android/app/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Sat Dec 21 13:15:18 CET 2019
```

Comment:
What do we need these gradle files for?
It should work without them, we didn't add any new libraries or similar?